src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -859,9 +859,7 @@ protected Void call() {
     // path to config file we will use for tabulation
     private final String configPath;
 
-    // function: TabulatorService
-    // purpose: constructor for Service object which runs a tabulation
-    // param: configPath path to config file to be tabulated
+    // constructor for Service object which runs a tabulation in the background
```

Comment:
I'd rather put this comment before the class declaration, not on the constructor